css/style.css

```diff
@@ -16,4 +35,83 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+* {
+    box-sizing: border-box;
+    font-size: 100%;
+    margin: 0;
+    border: 0;
+    padding: 0;
+    font-family: inherit;
+}
+
+body {
+    line-height: 1;
+}
+
+
+/* Nav section styling */
+
+.navigation {
+    width: 100%;
+    background-color: pink;
+    position: fixed;
+    z-index: 9999;
+}
+
+.navigation .links {
+    float: right;
+}
+
+.navigation .links li {
+    display: inline-block;
+    color: #838994;
+}
+
+.bold {
+    font-weight: bold;
+}
+
+.bold hover {
+    background-color: bisque;
+}
+
+.button {
+    width: 500px;
+    height: 200px;
+    padding: 40px 60px;
+}
+
+
+/* Introduction styling */
+
+.introduction {
+    background-image: url(../img/first-background.jpg);
+    height: 45.938rem;
+}
+
+
+/* Boxes styling */
+
+.features {
+    display: flex;
+    justify-content: center;
+}
+
+.caption {
+    position: relative;
+    top: 150px;
+    right: 160px;
+    width: 200px;
+    text-align: center;
```

Comment:
This rule is causing the `caption`s to appear outside of their parent. Remove this rule, move the captions inside their respective list items, and position them using flexbox.